cypress/integration/userCanCreateRecipe.spec.js

```diff
@@ -35,7 +41,7 @@ describe('Creates a recipe', () => {
       response: '{ "error_message": "Unable to create recipe." }',
```

Comment:
In the module requestRecipes.js, you are specifying two error responses on line 42. 
'error: error.response.data.error_message || error.message'
Is there a test for the second error message (for not attaching an image) or does it not need to be tested?

Reply:
It is a fallback if the image is bad. No need to test this.